Tools/WebKitTestRunner/InjectedBundle/AccessibilityUIElement.h

```diff
@@ -94,6 +94,7 @@ class AccessibilityUIElement : public JSWrappable {
 
     JSValueRef children(JSContextRef);
     RefPtr<AccessibilityUIElement> childAtIndex(unsigned);
+    RefPtr<AccessibilityUIElement> childAtIndexWithRemoteElementForTesting(unsigned);
```

Comment:
Is the `ForTesting` suffix necessary here? I feel like everything inside `AccessibilityUIElement` is implicitly for testing, because it cannot be used in any other way 🤔

Reply:
Ahh, gotcha. Well I'd be fine with either removing the suffix, or adding a comment explaining the name (as you've done here) 👍